ci/create-windows-image.ps1

```diff
@@ -1,278 +0,0 @@
-### This script is a guide for Improbable's internal build machine images.
```

Comment:
Why is this deleted?

Reply:
oops, that happened by accident. Will add it back in